README.md

```diff
@@ -112,10 +112,10 @@ tkn pipeline start operator-hosted-pipeline \
   --workspace name=pyxis-ssl-credentials,secret=operator-pipeline-api-certs \
   --workspace name=kubeconfig,secret=kubeconfig \
```

Comment:
This may or may not be removed upon a rebase but please ensure this line is also deleted.